docs/troubleshooting.md

```diff
@@ -22,3 +22,4 @@ discusses potential solutions to the same.
 | Repository changed 'Origin' and 'Label' error while running `apt-get update` command:  <br/>`E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Origin' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'`<br/>`E: Repository 'http://packages.cloud.google.com/apt gcsfuse-focal InRelease' changed its 'Label' value from 'gcsfuse-jessie' to 'namespaces/gcs-fuse-prod/repositories/gcsfuse-focal'`<br/>`N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details. ` | Use one of the following commands to upgrade to latest GCSFuse version<br/> `sudo apt-get update --allow-releaseinfo-change `<br/>OR<br/>`sudo apt update -y && sudo apt-get update`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | Unable to unmount or stop GCSFuse due to an error message like:`fusermount: failed to unmount: Device or resource busy` or `umount: /path/to/mountpoint: target is busy`.</br>This typically indicates active processes are using files or directories within the GCSFuse mount.                                                                                                                                                                                                                                                                                                                                      | Find the process ID of GCSFuse:<br/>`BUCKET=<Enter your bucket name>`</br>` MOUNT_POINT=<Enter your mount point>`</br>`PID=$(ps -aux &#124; grep "gcsfuse.*$BUCKET.*$MOUNT_POINT" &#124; grep -v grep &#124; tr -s ' ' &#124; cut -d' ' -f2)`</br>Kill the GCSFuse process:</br>`sudo kill -SIGKILL "$PID"`</br>Unmount GCSFuse</br>`fusermount -u $"MOUNT_POINT"`                                                                                                                                                                                                                                                     |
+| Unable to mount with the following error <br/> `daemonize.Run: readFromProcess: sub-process: Error while mounting gcsfuse: mountWithArgs: mountWithStorageHandle: Mount: mount: exec: "fusermount": executable file not found in $PATH` | Fuse package is not installed. It may throw this error. Run the following command to install fuse <br/> <code> sudo apt-get install fuse </code> |
```

Comment:
`sudo apt-get install fuse` for Debian/Ubuntu & `sudo yum install fuse` for RHEL/CentOs. 

CC' @ashmeenkaur

Reply:
for RHEL/CentOs/Rocky.

Reply:
Updated.